packages/components/Stack/src/Stack.tsx

```diff
@@ -21,7 +30,12 @@ const _styleKey = 'style';
 const render = (Slots: ISlots<IStackSlotProps>, renderData: IStackRenderData, ...children: React.ReactNode[]): JSX.Element => {
   const { gap, horizontal, wrap } = renderData.state!;
 
-  if (gap && gap > 0 && children) {
+  if (
+    gap &&
+    gap > 0 &&
+    children &&
+    (!global.hasOwnProperty('__jsiExecutorDescription') || global.__jsiExecutorDescription !== 'ChakraRuntime')
```

Comment:
TODO: Need to add a comment for this

Also fix the experimental Stack.

Reply:
`__jsiExecutorDescription` would just be `undefined` if it isn't set, so you can drop the first check:

```suggestion
    global.__jsiExecutorDescription !== 'ChakraRuntime'
```

Reply:
Thanks, change has been made.